playground/public/assets/export-db.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+use Drupal\Core\Cache\Cache;
+use Drupal\Core\DrupalKernel;
+use Symfony\Component\HttpFoundation\Request;
+
+$stdErr = fopen('php://stderr', 'w');
+
+set_error_handler(static function(...$args) use($stdErr) {
+    fwrite($stdErr, print_r($args,1));
+});
+
+$flavor = file_get_contents('/config/flavor.txt');
+unlink('/config/flavor.txt');
+
+$autoloader = require "/persist/$flavor/vendor/autoload.php";
+
+$request = Request::createFromGlobals();
+$kernel = DrupalKernel::createFromRequest($request, $autoloader, 'prod');
+chdir($kernel->getAppRoot());
+$kernel->boot();
+
+foreach (Cache::getBins() as $cache_backend) {
+    $cache_backend->deleteAll();
+}
+\Drupal::service('plugin.cache_clearer')->clearCachedDefinitions();
+$kernel->invalidateContainer();
+
+$sql = 'PRAGMA foreign_keys=OFF;' . PHP_EOL;
+$sql .= 'BEGIN TRANSACTION;' . PHP_EOL;
+
+// adapted from https://github.com/ephestione/php-sqlite-dump/blob/master/sqlite_dump.php
+$database = \Drupal::database();
+$tables = $database->query('SELECT [name], [sql] FROM {sqlite_master} WHERE [type] = "table" AND [name] NOT LIKE "sqlite_%" ORDER BY name');
+foreach ($tables as $table) {
+    $sql .= str_replace('CREATE TABLE ', 'CREATE TABLE IF NOT EXISTS ', $table->sql) . ';' . PHP_EOL;
+
+    $columns = array_map(
+        static fn (object $row) => "`{$row->name}`",
+        $database->query("PRAGMA table_info({$table->name})")->fetchAll()
+    );
+    $columns = implode(', ', $columns);
+    $sql .= PHP_EOL;
+
+    $rows = $database->select($table->name)->fields($table->name)->execute()->fetchAll(\PDO::FETCH_ASSOC);
+    foreach ($rows as $row) {
+        $values = implode(
+            ', ',
+            array: array_map(
+                static function ($value) {
+                    if ($value === NULL) {
+                        return 'NULL';
+                    }
+                    if ($value === '') {
+                        return  "''";
+                    }
+                    if (is_numeric($value)) {
+                        return $value;
+                    }
+                    if (is_string($value)) {
+                        // @todo: this str_contains isn't working correctly
+                        if (str_contains(PHP_EOL, $value)) {
+                            $value = str_replace([PHP_EOL, "\r", "\n", "\r\n"], "\\n", $value);
+                            $value = "replace($value,'\\n',char(10))";
```

Comment:
Fix this. Example is config values of user emails when registered, password reset, etc